src/main/java/carpet/commands/DrawCommand.java

```diff
@@ -109,30 +205,25 @@ private static int drawCircle(ServerCommandSource source, BlockPos pos, double r
                     }
 
                     if (!solid) {
-                        if (lengthSq(nextXn, yn, zn) <= 1 && lengthSq(xn, nextYn, zn) <= 1 && lengthSq(xn, yn, nextZn) <= 1) {
+                        if (lengthSq(nextXn, yn, zn) <= 1 && lengthSq(xn, nextYn, zn) <= 1
+                                && lengthSq(xn, yn, nextZn) <= 1) {
                             continue;
                         }
                     }
 
                     CarpetSettings.impendingFillSkipUpdates = !CarpetSettings.fillUpdates;
-                    for (int xmod = -1; xmod < 2; xmod+= 2)
```

Comment:
Please undo your egyptian braces

Reply:
could you please be more specific?

Reply:
ok, i googled egyptian braces. This programming jargon is so weird compared to science lab stuff.
I'll remove the egyptian braces that I added into you methods